addons/point_of_sale/tests/test_pos_products_with_tax.py

```diff
@@ -0,0 +1,348 @@
+import odoo
```

Comment:
To avoid having to create multiple times sessions, and orders, you should put the 'multiple_sale_accounts', 'multiple_receivable', 'fiscal_positon' and 'products_with_tax' together, valid all those orders, and check the entries at the end. Because we don't want to have too much time on runbot, so if we can test multiple features in 3 tests it'll be faster than 12 tests where you have to crete products, create session, create invoices, and close session.

Reply:
I would disagree on this. Making the tests separate reduces the cognitive loads to the developer. The module should be maintained and making the test scenarios independent from each other makes the development easier -- by making it easier to isolate the problem. If a feature implementation or a bug fix is done, and the testing leads to failure in one test, attacking this isolated failure will be easier than the analyzing what is wrong in the combined test scenario.

Nevertheless, I will also create a test that combines different scenarios. I think runbot can handle the isolated tests.